service/static/js/rest_api.js

```diff
@@ -504,4 +504,12 @@ $(function () {
         clear_form_data()
     });
 
+    // ****************************************
+    // Links to swagger api docs
+    // ****************************************
+
+    $("#api-btn").click(function () {
```

Comment:
I think we can remove this button as well.

Reply:
ok! I will do it rn!